agent_deploy/kubernetes/sysdig-image-analyzer-daemonset.yaml

```diff
@@ -30,6 +30,10 @@ spec:
       - name: var-lib-containers-vol
         hostPath:
           path: /var/lib/containers
+      # Needed for some IBM OpenShift clusters which symlink /var/run/containers/storage to contents of /var/data by default
+      - name: vardata-vol
+        hostPath:
+          path: /var/data
```

Comment:
what happens if this path does not exists? Has that been tested?

Reply:
Nothing happens if the directory is missing. To validate this we have ran integration and smoke tests with this DS against regular Kops, OpenShift and GKE clusters via regression tests by @franco-nonne